docs/concepts/storage/persistent-volumes.md

```diff
@@ -573,68 +573,51 @@ parameters:
 
 #### vSphere
 
-1. Create a persistent volume with a user specified disk format.
+   1. Create a StorageClass with a user specified disk format.
 
-```yaml
-kind: StorageClass
-apiVersion: storage.k8s.io/v1
-metadata:
-  name: fast
-provisioner: kubernetes.io/vsphere-volume
-parameters:
-  diskformat: zeroedthick
-```
+       ```yaml
+       kind: StorageClass
+       apiVersion: storage.k8s.io/v1
+       metadata:
+         name: fast
+       provisioner: kubernetes.io/vsphere-volume
+       parameters:
+         diskformat: zeroedthick
+       ```
 
--   `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
+       - `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
 
-2. Create a persistent volume with a disk format on a user specified datastore.
+    2. Create a persistent volume with a disk format on a user specified datastore.
+  
+        ```yaml
+        kind: StorageClass
+        apiVersion: storage.k8s.io/v1
+        metadata:
+          name: fast
+        provisioner: kubernetes.io/vsphere-volume
+        parameters:
+            diskformat: zeroedthick
+            datastore: VSANDatastore
+        ```
 
-```yaml
-kind: StorageClass
-apiVersion: storage.k8s.io/v1
-metadata:
-  name: fast
-provisioner: kubernetes.io/vsphere-volume
-parameters:
-    diskformat: zeroedthick
-    datastore: VSANDatastore
-```
+       -   `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
```

Comment:
Are these unordered bullets under the second bullet point? You probably need to fix the formatting for this.